Resources/public/js/views/fields/ez-date-editview.js

```diff
@@ -55,6 +55,19 @@ YUI.add('ez-date-editview', function (Y) {
         },
 
         /**
+         * Checks whether the field contains a valid timestamp.
+         *
+         * @method _containsValidTimestamp
+         * @protected
+         * @return {Boolean}
+         */
+        _containsValidTimestamp: function () {
```

Comment:
After discussing with @dpobel it's not the exact same behavior. Forget about the previous comment.